src/site/content/en/learn/pwa/web-app-manifest/index.md

```diff
@@ -1,11 +1,310 @@
 ---
 title: Web app manifest
 description: >
-  The Web App Manifest is mandatory to every PWA. In this chapter, you will see all the meta data that can be set to define how your app will work.
+  The Web App Manifest is a JSON file that defines how the PWA should be treated as an installed application, including the look and feel and basic behavior within the operating system.
 authors:
   - firt
-date: 2021-11-03
-placeholder: true
+date: 2022-01-10
 ---
 
-Coming soon!
+The manifest is a file you create as a guide telling the browser how you want your web content to display as an app in the operating system. The manifest can include basic information such as the app's name, icon, and theme color, advanced preferences, such as desired orientation and app shortcuts, and catalog metadata, such as screenshots.
+
+{% Aside 'warning' %}
+Having a Web App manifest present in your PWA is required to pass the PWA installability criteria in every browser.
+{% endAside %}
+
+Each PWA should include a single manifest per application, typically hosted in the root folder, and linked on all HTML pages your PWA can be installed from. Its official extension is `.webmanifest`, so you could name your manifest something like `app.webmanifest`.
+
+{% Aside %}
+While the recommended extension is `.webmanifest`, any filename will work as long as it's delivered with the `application/manifest+json` content type or another JSON-valid content type, such as `text/json`. As such, many PWAs use `manifest.json` instead.
+{% endAside %}
+
+## Adding a Web App Manifest to your PWA
+
+To create a Web App Manifest, first make a text file with a JSON object that contains at least a `name` field with a string value:
+{% Label %}app.webmanifest:{% endLabel %}
+
+```json
+{
+   "name": "My First Application"
+}
+```
+
+But creating the file is not enough, the browser needs to know it exists, too.
+
+### Linking to your manifest
+
+To make the browser aware of your Web Application Manifest, you need to link it to your PWA using a `<link>` HTML element and the `rel` attribute set to `manifest` on all of your PWA's HTML pages, similar to how you link a CSS stylesheet to a document.
+
+{% Label %}index.html:{% endLabel %}
+
+```html/3
+<html lang="en">
+  <title>This is my first PWA</title>
+  <link rel="manifest" href="/app.webmanifest">
+```
+
+{% Aside 'caution' %}
+If your PWA includes multiple HTML pages, make sure to link its manifest to all of them.
+{% endAside %}
+
+### Debugging the manifest
+
+To ensure the manifest is set up correctly, you can use Inspector in Firefox and DevTools in every Chromium-based browser. 
+
+{% Aside 'gotchas' %}
+If you see a message saying that the browser detected no manifest and you've included a `<link>` to it correctly, check to make sure you can load the manifest in the browser and that its content type is correctly set. You can also try renaming it to `manifest.json`.
```

Comment:
Ping!

Reply:
So I tracked down the original sentence, it was this one: "if you can load the manifest URL in the browser or if the content MIME type is correctly set in your server. In case you don't have experience dealing with content types, try to use manifest.json as the name of your Web App Manifest." 

So the reason the wanted to offer the option to rename the manifest to something.json (and they chose manifest, because it is the most used) is that a web server might not be set up properly to serve the manifest as json but is likely to serve json properly.

Reply:
That original seems better. Can we use that instead? I don't think we should assert without explanation.